packages/postmessage/index.js

```diff
@@ -1,7 +1,7 @@
 let seq = 1
 let reqMap = new Map()
 const RESPONSE = '__RESPONSE__'
-const TRANSFERABLE = '__transferable'
+const TRANSFERABLE = Symbol.for('__transferable__')
```

Comment:
Why did this change?

Reply:
symbol does not get serialized when you attach value to an object where key is a symbol. So this piggyback of transferable information is less intrusive.